runtimes/build-local.sh

```diff
@@ -0,0 +1,4 @@
+echo 'Starting local build...'
+
+echo 'Node 17...'
+docker build -t open-runtimes/node:17.0 ./runtimes/node-17.0
```

Comment:
I think these paths need to be fixed. Keep in mind the folder structure

Reply:
Resolved by other suggestions